Documentation/api-reference.md

```diff
@@ -134,23 +134,38 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
 ##### Example: sendEvent
 
 ```brightscript
-  m.aepSdk.sendEvent({
-    "eventType": "commerce.orderPlaced",
+  data = {
+    "xdm" : {
+      "eventType": "commerce.orderPlaced",
       "commerce": {
         .....
       }
-  })
+    },
+    "data" : {
+      "customKey" : "customValue"
+    }
+  }
```

Comment:
can we separate these two xdm and free-form non-xdm examples in two simple examples - see a similar example from the web sdk https://experienceleague.adobe.com/docs/experience-platform/edge/fundamentals/tracking-events.html